core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java

```diff
@@ -428,8 +431,9 @@ public boolean hasNext() {
         while (mountId == -1) {
           try {
             syncPointPath = mEntry.getPath();
-            MountTable.Resolution resolution = mMountTable.resolve(mEntry);
-            mountId = resolution.getMountId();
+            String mountPoint = mMountTable.getMountPoint(mEntry);
```

Comment:
What is the difference from before? How does the new code avoid UFS access?

Reply:
`MountTable::resolve()` will actually creates a UFS client to query the UFS path while the new code only grabs the `MountInfo`. This function only need the mount id so resolution is not required.